Xamarin.Forms.Controls.Issues/Xamarin.Forms.Controls.Issues.Shared/Issue9631.cs

```diff
@@ -0,0 +1,108 @@
+﻿using System.Collections.ObjectModel;
+using Xamarin.Forms.CustomAttributes;
+using Xamarin.Forms.Internals;
+using System;
+using Xamarin.Forms.Xaml;
+using System.Collections.Generic;
+
+#if UITEST
+using Xamarin.UITest;
+using Xamarin.UITest.Queries;
+using NUnit.Framework;
+using Xamarin.Forms.Core.UITests;
+using System.Linq;
+#endif
+
+namespace Xamarin.Forms.Controls.Issues
+{
+#if UITEST
+	[Category(UITestCategories.CollectionView)]
+#endif
+#if APP
+	[XamlCompilation(XamlCompilationOptions.Compile)]
+#endif
+	[Preserve(AllMembers = true)]
+	[Issue(IssueTracker.Github, 9631, "[CollectionView] How to scroll in Horizontal CollectionView",
+		PlatformAffected.UWP)]
+	public partial class Issue9631 : TestContentPage
+	{
+
+		protected override void Init()
+		{
+			CollectionView collectionView = new CollectionView { ItemsLayout = LinearItemsLayout.Horizontal, };
+			collectionView.ItemTemplate = new DataTemplate(() =>
+			{
+				var stacklayout = new StackLayout
+				{
+					Orientation = StackOrientation.Vertical,
+					Spacing = 5,
+					Padding = 10,
+					BackgroundColor = Color.Beige
+				};
+
+
+				Image image = new Image { Aspect = Aspect.AspectFill, HeightRequest = 60, WidthRequest = 60 };
+				image.SetBinding(Image.SourceProperty, "Source");
+
+				Label nameLabel = new Label { FontAttributes = FontAttributes.Bold };
+				nameLabel.SetBinding(Label.TextProperty, "Text");
+
+				nameLabel.SetBinding(Label.AutomationIdProperty, "AutomationId");
+
+
+				stacklayout.Children.Add(image);
+				stacklayout.Children.Add(nameLabel);
+				 
+				return stacklayout;
+			});
+			Content = collectionView;
+			collectionView.SetBinding(ItemsView.ItemsSourceProperty, "Items");
+ 
+		
+			BindingContext = new ViewModel9631();
+		}
+
+
+
+		[Preserve(AllMembers = true)]
+		public class ViewModel9631
+		{
+			public ObservableCollection<Model9631> Items { get; set; }
+
+			public ViewModel9631()
+			{
+				var collection = new ObservableCollection<Model9631>();
+				var pageSize = 50;
+
+				for (var i = 0; i < pageSize; i++)
+				{
+					collection.Add(new Model9631
+					{
+						Text = "Image" + i,
+						Source = i % 2 == 0
+							? "https://upload.wikimedia.org/wikipedia/commons/thumb/5/5d/Kamchatka_Brown_Bear_near_Dvuhyurtochnoe_on_2015-07-23.jpg/320px-Kamchatka_Brown_Bear_near_Dvuhyurtochnoe_on_2015-07-23.jpg"
```

Comment:
Let's not depend on resources outside of the repository, please!

Reply:
this is not from me. I have copied one of the test case and reused it. it exist in several issue test cases.
https://github.com/xamarin/Xamarin.Forms/search?q=https%3A%2F%2Fupload.wikimedia.org%2Fwikipedia%2Fcommons%2Fthumb&unscoped_q=https%3A%2F%2Fupload.wikimedia.org%2Fwikipedia%2Fcommons%2Fthumb

Reply:
Fair enough :)

Reply:
Despite that i have changed it now using local icon.